instagram-page-react/src/pages/profilePage/organisms/posts/actions/setPosts.action.js

```diff
@@ -0,0 +1,10 @@
+import ACTION_TYPES from "../reducers/actionTypes";
```

Comment:
This file should be moved to `pages/profilePage/actions/profilePage.posts`

Reply:
Clashes with the other comment, following that have kept in the reducers folder and renamed it to `profilePage.posts`